dolphinscheduler-task-plugin/dolphinscheduler-task-jupyter/src/main/java/org/apache/dolphinscheduler/plugin/task/jupyter/JupyterPropertyReader.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.task.jupyter;
+
+import org.apache.dolphinscheduler.spi.utils.PropertyUtils;
+
+// this is a wrapper class to decouple untestable static method
+
+public class JupyterPropertyReader {
+
+    protected String readProperty(final String key) {
```

Comment:
We discussed this in another issue during yesterday's community conference. `Mockito.mockStatic` does not work for `PropertyUtils`. We think the best way is to fix `PropertyUtils` instead of adding a wrapper here. cc @ruanwenjun

Reply:
@caishunfeng I just removed the `JupyterPropertyReader` class in the latest commit and found a more graceful way to deal with `PropertyUtils` in UT.

Reply:
Oooops, I forgot to delete it. Have removed it in the latest commit. Thanks!